core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -96,7 +96,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
 
   val startTime = System.currentTimeMillis()
 
-  private val stopped: AtomicBoolean = new AtomicBoolean(false)
+  private[spark] val stopped: AtomicBoolean = new AtomicBoolean(false)
```

Comment:
I think he means expose it and make it `@DeveloperApi` instead of `private[spark]`